rules/signing.bzl

```diff
@@ -170,12 +216,52 @@ def _local_sign(ctx, opentitantool, digest, rsa_key, spxmsg = None, spx_key = No
                 spxmsg.path,
                 spx_key.file.path,
             ],
-            executable = opentitantool,
+            executable = tool.tool,
             mnemonic = "LocalSpxSign",
         )
 
     return rsa_sig, spx_sig
 
+def _hsmtool_sign(ctx, tool, digest, rsa_key, spxmsg = None, spx_key = None, profile = None):
+    """Sign a digest with a local on-disk RSA private key.
```

Comment:
nit: Can we update this to read "Sign a digest with an RSA key"? We can also mention that this mode of operation supports signing via Tokens and HSMs.